db/seed/data_migration_0005.sql

```diff
@@ -0,0 +1,7 @@
+
+BEGIN TRANSACTION;
+
+UPDATE cluster_namespace
+  SET quota_cpu = 'small', quota_memory = 'small', quota_storage = 'small';
+
+END TRANSACTION;
```

Comment:
Nit: As above, no newline
```suggestion
END TRANSACTION;

```